github/template.go

```diff
@@ -0,0 +1,77 @@
+package github
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/github/hub/utils"
+)
+
+const (
+	pullRequestTempalte = "PULL_REQUEST_TEMPLATE"
+	issueTempalte       = "ISSUE_TEMPLATE"
+	githubTemplateDir   = ".github"
+)
+
+func GetPullRequestTempalte() string {
```

Comment:
There's a typo here. Should be `GetPullRequestTemplate `

Reply:
![Feature](https://user-images.githubusercontent.com/51739397/129476928-fbd7aa54-d145-44de-a736-ca7527282408.png)

Reply:
hhhh
![Feature](https://user-images.githubusercontent.com/51739397/129477108-936b5893-e95d-438d-8cd5-aa5511d9255d.png)